tukaan/_tcl.py

```diff
@@ -178,28 +214,36 @@ def from_(return_type: type[object], value: str | tk.Tcl_Obj) -> Any:  # noqa: C
                 items_len = len(sequence)
                 if len(return_type) != items_len:
                     return_type *= items_len
-                return tuple(map(Tcl.from_, return_type, sequence))
+                return tuple(Tcl.from_(return_type, elem) for elem in sequence)
```

Comment:
```suggestion
                return tuple(Tcl.from_(type_, elem) for type_, elem in zip(return_type, sequence))
```
It would be correct this way, but using `map` here is much cleaner